### PR TITLE
[Feature][E2E] Restore security center e2e test cases in ui-next

### LINKED ISSUE #8727: [Feature][E2E] Restore security center e2e test cases in ui-next
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/dolphinscheduler/issues?q=is%3Aissue) and found no similar feature requirement.

### Description

Restore resource center e2e test cases in ui-next

### Use case

_No response_

### Related issues

_No response_

### Are you willing to submit a PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by SbloodyS**
## Purpose of the pull request

This pr will close #8727

Temporary comment out ```org/apache/dolphinscheduler/e2e/cases/UserE2ETest.java#testDeleteUser``` until the front end fixed the problem of deleting styles to restore.

### CODE REVIEW

**Review by songjianet — DISMISSED** [review]
LGTM

File: dolphinscheduler-e2e/dolphinscheduler-e2e-case/src/test/java/org/apache/dolphinscheduler/e2e/cases/FileManageE2ETest.java [hunk 1]
```diff
@@ -84,20 +84,20 @@
...
+//        await().untilAsserted(() -> assertThat(tenantPage.tenantList())
+//                .as("Tenant list should contain newly-created tenant")
+//                .extracting(WebElement::getText)
+//                .anyMatch(it -> it.contains(tenant)));
+//
+//        tenantPage.goToNav(SecurityPage.class)
+//            .goToTab(UserPage.class)
+//            .update(user, user, password, email, phone)
+//            .goToNav(ResourcePage.class)
+//            .goToTab(FileManagePage.class);
```
Comment (kezhenxu94) [inline dolphinscheduler-e2e/dolphinscheduler-e2e-case/src/test/java/org/apache/dolphinscheduler/e2e/cases/FileManageE2ETest.java:100]: These should be reverted?

File: dolphinscheduler-e2e/dolphinscheduler-e2e-case/src/test/java/org/apache/dolphinscheduler/e2e/cases/FunctionManageE2ETest.java [hunk 2]
```diff
@@ -83,28 +83,28 @@
...
+//            .goToTab(UserPage.class)
+//            .update(user, user, password, email, phone)
+//            .goToNav(ResourcePage.class)
+//            .goToTab(UdfManagePage.class)
+//            .uploadFile(testUploadUdfFilePath.toFile().getAbsolutePath());
+//
+//        new WebDriverWait(browser, 10).until(ExpectedConditions.invisibilityOfElementLocated(By.id("fileUpdateDialog")));
+//
+//        udfManagePage.goToNav(ResourcePage.class)
+//            .goToTab(FunctionManagePage.class);
```
Comment (kezhenxu94) [inline dolphinscheduler-e2e/dolphinscheduler-e2e-case/src/test/java/org/apache/dolphinscheduler/e2e/cases/FunctionManageE2ETest.java:107]: Should be reverted?

File: dolphinscheduler-e2e/dolphinscheduler-e2e-case/src/test/java/org/apache/dolphinscheduler/e2e/cases/QueueE2ETest.java [hunk 3]
```diff
@@ -86,7 +86,8 @@ void testCreateDuplicateQueue() {
     @Test
     @Order(30)
     void testEditQueue() {
-        final QueuePage page = new QueuePage(browser);
+        QueuePage page = new QueuePage(browser);
```
Comment (kezhenxu94) [inline dolphinscheduler-e2e/dolphinscheduler-e2e-case/src/test/java/org/apache/dolphinscheduler/e2e/cases/QueueE2ETest.java:89]: Should be reverted?

```suggestion
        final QueuePage page = new QueuePage(browser);
```

File: dolphinscheduler-e2e/dolphinscheduler-e2e-case/src/test/java/org/apache/dolphinscheduler/e2e/cases/TokenE2ETest.java [hunk 4]
```diff
@@ -52,8 +52,8 @@ public static void setup() {
     @Test
     @Order(10)
     void testCreateToken() {
-        final TokenPage page = new TokenPage(browser);
-        page.create();
+        TokenPage page = new TokenPage(browser);
```
Comment (kezhenxu94) [inline dolphinscheduler-e2e/dolphinscheduler-e2e-case/src/test/java/org/apache/dolphinscheduler/e2e/cases/TokenE2ETest.java:55]: Why remove these `final`?

File: dolphinscheduler-e2e/dolphinscheduler-e2e-case/src/test/java/org/apache/dolphinscheduler/e2e/cases/UserE2ETest.java [hunk 5]
```diff
@@ -126,21 +127,21 @@ void testEditUser() {
...
+//        await().untilAsserted(() -> {
+//            browser.navigate().refresh();
+//
+//            assertThat(
+//                page.userList()
+//            ).noneMatch(
+//                it -> it.getText().contains(user) || it.getText().contains(editUser)
+//            );
+//        });
+//    }
```
Comment (kezhenxu94) [inline dolphinscheduler-e2e/dolphinscheduler-e2e-case/src/test/java/org/apache/dolphinscheduler/e2e/cases/UserE2ETest.java:146]: Can we uncomment these lines?

**Review by kezhenxu94 — COMMENTED** [review]
@SbloodyS can you overhaul the commented lines? They are not executed in this PR so we don't need to comment them, no?

**Review by kezhenxu94 — COMMENTED** [review]
Otherwise LGTM if tests passed

**Review by kezhenxu94 — APPROVED** [review]
Thank you very much!

**Pull request merged**